src/Microsoft.DocAsCode.MarkdigEngine/Validation/ValidationExtension.cs

```diff
@@ -25,6 +28,17 @@ public void Setup(MarkdownPipelineBuilder pipeline)
 
             pipeline.DocumentProcessed += document =>
             {
+                if (InclusionContext.IsInclude 
```

Comment:
You can extract a method for this delegate, then test can also use this method.  `public void Validate(MarkdownDocument document)`

Reply:
Thanks for the suggestion, done with extraction.